docs/conf.py

```diff
@@ -50,6 +50,7 @@
     "attributetable",
     "resourcelinks",
     "nitpick_file_ignorer",
+    "notfound.extension",
```

Comment:
This should be moved below `hoverxref.extension`, so that all custom extensions are at the end

Reply:
https://github.com/DisnakeDev/disnake/pull/797/commits/46352b863aebe7bb590c570cda56ae9702cb4ad2